torchci/test/drciBot.test.ts

```diff
@@ -33,6 +35,9 @@ describe("verify-drci-functionality", () => {
     payload["repository"]["owner"]["login"] = OWNER;
     payload["repository"]["name"] = REPO;
 
+    const rockset = nock("https://api.rs2.usw2.rockset.com")
```

Comment:
Do these tests intentionally not check that scope.isDone() or is that a bug

Reply:
unsure but added anyways